components/ui/Footer.js

```diff
@@ -12,43 +16,36 @@ export const Footer = () => {
                 <p>Explore the universe and beyond. Your journey to the stars starts here.</p>
                 <p>&copy; 2024 Galactica. All rights reserved.</p>
             </div>
-            {/* TASK - React 1 week 2 */}
-            {/* Create a new List for the Pages */}
-            {/* We need to use the <Link /> component here */}
-            {/* <div className={styles.pages}>
-        <h3>Pages</h3>
-        <ul>
-          <li> <Link/> </li>
-          ...
-        </ul>
-      </div> */}
-            {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
 
-            {/* TASK - React 1 week 1 */}
-            {/* Add a new list item for LINKEDIN */}
-            <div className={styles.footerLinks}>
-                <h3>Follow us</h3>
-                <ul className={styles.footerList}>
-                    <li>
-                        <a href="https://facebook.com">Facebook</a>
-                    </li>
+            <div className={styles.pages}>
+                <h3>Pages</h3>
+                <ul>
                     <li>
-                        <a href="https://instagram.com">Instagram</a>
+                        <Link href="/">Home</Link>
                     </li>
                     <li>
-                        <a href="https://linkedin.com">LinkedIn</a>
+                        <Link href="/about_us">About Us</Link>
                     </li>
                     <li>
-                        <a href="https://tiktok.com">Tiktok</a>
+                        <Link href="/destination">Destination</Link>
                     </li>
                     <li>
-                        <a href="https://google.com">On the streets at night</a>
+                        <Link href="/nasa_collaboration">NASA Collaboration</Link>
                     </li>
-                    {/* TASK - React 1 week 2 */}
-                    {/* Create a <SocialMediaItem /> component and replace all of the list items! */}
-                    {/* it should accept the following props */}
-                    {/* url, title, icon */}
-                    {/* For the icons, you can download 1-2 social media icons for testing and put it in the /public/socialmedia/ folder */}
+                </ul>
+            </div>
+
+            <div className={styles.footerLinks}>
+                <h3>Follow us</h3>
+                <ul className={styles.footerList}>
+                    {socialMediasData.map((media, index) => (
+                        <SocialMediaItem
+                            key={index}
```

Comment:
We already used the `key={index}` in many other places, and this can cause some errors. So to make it unique it could be:

```
key={`social-${index}`}
```